src/schematron/preprocessor.xsl

```diff
@@ -1,24 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xsl:stylesheet exclude-result-prefixes="#all" version="3.0"
-	xmlns:x="http://www.jenitennison.com/xslt/xspec" xmlns:xs="http://www.w3.org/2001/XMLSchema"
-	xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
-	<xsl:variable as="map(xs:string, item())" name="x:schematron-preprocessor" select="
+<xsl:stylesheet
+        exclude-result-prefixes="#all"
+        version="3.0"
+        xmlns:x="http://www.jenitennison.com/xslt/xspec"
+        xmlns:xs="http://www.w3.org/2001/XMLSchema"
+        xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+    <xsl:variable name="sbu" select="resolve-uri('../../lib/', static-base-uri())" static="yes"/>
+    <xsl:variable as="map(xs:string, item())" name="x:schematron-preprocessor"
+	        select="
```

Comment:
```suggestion
<xsl:stylesheet exclude-result-prefixes="#all" version="3.0"
	xmlns:x="http://www.jenitennison.com/xslt/xspec" xmlns:xs="http://www.w3.org/2001/XMLSchema"
	xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
	<xsl:variable name="sbu" select="resolve-uri('../../lib/', static-base-uri())" static="yes"/>
	<xsl:variable as="map(xs:string, item())" name="x:schematron-preprocessor" select="
```

My suggestion merely formats the code differently, to simplify the diffs of this PR and also use the same kind of indentation (tab vs. space character) throughout the file.

Reply:
Applied on #65d2c83d.